src/render/dom/index.ts

```diff
@@ -104,9 +105,13 @@ export const motion = /*@__PURE__*/ createMotionProxy(allMotionFeatures)
  *
  * @public
  */
-export function createDomMotionComponent(key: string) {
+export function createDomMotionComponent(key: HTMLElements) {
+    type motionTypes = HTMLMotionComponents & SVGMotionComponents
```

Comment:
I think these types can be defined outside the function? And should start with a capital letter.

Reply:
Sure it is, i tried to keep the same pattern for internal types you adopted in `createMotionProxy` by creating function scoped types
Where should I create these types ? `src/render/dom/types.ts` ?
Also seems good to have a single way for reference these types and adapt the types inside createMotionProxy (possibly)

Reply:
You can create them in this file, I think I only did that with createMotionproxy as the types themselves relied on incoming arguments (otherwise I messed up 😅)